ACMILabs/spotlights
# #82 Update npm packages

Resolves #82

Update npm packages.

### Acceptance Criteria
- [x] `npm update`

### Relevant design files
* None

### Testing instructions
1. See this build passes
2. Run `npm audit` and note no vulnerabilities

### DoD
For requester to complete:
~- [ ] New logic has been documented~
~- [ ] New logic has appropriate unit tests~
~- [ ] Deployment / migration instruction have been updated if required~
